src/fixturelibrary.ts

```diff
@@ -127,6 +149,25 @@ export class FixtureLibrary {
     return fixture;
   }
 
+  public async getMakeModelIndex(): Promise<MakeModel[]> {
+    const items = await this.fixtureIndex.getIndex();
+    const manufacturers = this.fixtureIndex.getManufacturers();
+    // eslint-disable-next-line no-async-promise-executor
+    return await Promise.all(Object.keys(items).map((key) => new Promise(async (resolve) => {
+      const fixture = await this.getFixture(key);
+      const path = items[key]?.path;
+      // eslint-disable-next-line no-useless-escape
+      const makePath = path?.match(/^([^\/]+)/)?.[0]; // Gets make
```

Comment:
```suggestion
      const makePath = path?.split('/')?.[0]; // Gets make
```

☝️ Is that what this regex is doing?